src/main/java/com/salesforce/Main.java

```diff
@@ -74,6 +75,16 @@ public static void main(String[] args) throws Exception {
             System.exit(4);
         }
         String topicPrefix = settings.getProperty("default_topic_prefix");
+
+        if ( topicPrefix.contentEquals("use_hostname") ) {
+            try {
+                topicPrefix = java.net.InetAddress.getLocalHost().getHostName();
+            } catch ( UnknownHostException e ) {
+                // dns is broken :(
```

Comment:
Maybe `log.warn`?

Why not just have the process error and exit out completely?